.github/workflows/stale.yaml

```diff
@@ -0,0 +1,24 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 7 * * 1-5' # every morning at 7 AM, only work days
+
+permissions:
+  contents: write
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-stale: 60
+          stale-issue-message: "This issue has been marked as stale due to the lack of updates in the last 60 days"
+          days-before-close: 7
+          close-issue-message: "This issue has been closed due to the lack of updates"
```

Comment:
```suggestion
          close-issue-message: "This issue has been closed due to the lack of updates"
          close-issue-reason: 'not_planned'
```

Reply:
This will be the message that will appear as a comment in the issue, why would you prefer it that way?

Reply:
Just to know that the issue has not been worked on.
I am fine with keeping just the message though.